utils/transmission-create.1

```diff
@@ -7,8 +7,8 @@
 .Sh SYNOPSIS
 .Bk -words
 .Nm
-.Op Fl h 
-.Op Fl p 
+.Op Fl p
+.Op Fl h
```

Comment:
Why the reversed order here? :)

Reply:
I don't remember doing this. Probably an accident swapping lines.